webapp/channels/src/i18n/en.json

```diff
@@ -5043,7 +5043,6 @@
   "suggestion.mention.all": "Notifies everyone in this channel",
   "suggestion.mention.channel": "Notifies everyone in this channel",
   "suggestion.mention.channels": "My Channels",
-  "suggestion.mention.groups": "Group Mentions",
```

Comment:
The two strings that were removed from here don't seem to actually have been used. They're very similar to other strings (this one to `suggestion.search.groups` and the other to `suggestion.mention.unread`), so I think they were just added by mistake